public/video-ui/src/components/ScheduleRecap/ScheduleRecap.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+
+const ScheduleRecap = ({ scheduledLaunch, embargo, hasPreventedPublication }) => (
+  !hasPreventedPublication ?
+  <div className="topbar__launch-label">
+    <div>
+      <span className="scheduledSummary--scheduledLaunch">
+        {'Scheduled: '}
+      </span>
+      {scheduledLaunch
+        ? moment(scheduledLaunch).format('Do MMM YYYY HH:mm')
```

Comment:
This format seems to be used in quite a few places as well - it could also live in a constants file.

Reply:
I think I'd rather keep this explicit, because in `VideoItem` we are using a slightly different format (`'D MMM HH:mm'` vs `'Do MMM HH:mm'`), and now it's easy to spot the difference

Reply:
We could make them both constants of course can keep them like this as well